README.md

```diff
@@ -63,15 +63,24 @@ librosa (0.x.x, /path/to/librosa)
 
 ### Hints for the Installation
 
-#### audioread
+`librosa` uses `soundfile` and `audioread` to load audio files.
+Note that `soundfile` does not currently support MP3, which will cause librosa to
+fall back on the `audioread` library.
 
-*Note that `audioread` needs at least one of the programs to work properly.*
+#### soundfile
+
+If you're using `conda` to install librosa, then all audio coding dependencies (except MP3) will be handled automatically.
```

Comment:
I have to double check this, but the last time I checked, the conda-forge builds of libsndfile didn't came with flac support. See issue [here](https://github.com/conda-forge/libsndfile-feedstock/issues/2)

Reply:
Hmmm... that's deeply unfortunate.  I'll find a way to reword this to more accurately reflect the sad state of codec packaging.